onnx/onnx.proto

```diff
@@ -52,17 +52,52 @@ enum Version {
   // version that the  graph is generated from. This helps us set up version
   // control. We should use version as
   //     xx(major) - xx(minor) - xxxx(bugfix)
-  // and we are starting with 00000001.
-  IR_VERSION = 00000001;
+  // and we are starting with 00000001, which was the
+  //  version we published on Oct 10, 2017.
+  IR_VERSION_2017_10_10 = 0x00000001;
+
+  // IR_VERSION 1.0.1 (this proto file) contains the following IR format changes
+  // - Added type discriminator to AttributeProto to support proto3 users
+
+  IR_VERSION = 0x00010001;
```

Comment:
the version format was xx.xx.xxxx, so the version should be updated as 0x00010000 (0.1.0), no?

Reply:
I'm assuming we are parsing little-endian based on the initial value we used (000000001), which I assumed was 1.0.0.  This PR simply bumps the version to 1.0.1, which is 0x00010001

Reply:
I think the current version is 0.0.1 :)

Reply:
I think the current version is 0.0.1 :)

Reply:
I think the current version is 0.0.1 :)

Reply:
We resolved offline that the spec in master is at 0.0.1.

I'm writing a proposal (that will get checked in as part of the ONNX spec) for how we will deal with all of this so we don't have to have this discussion multiple times :-)